--- 2012BOS.EVA	2014-11-19 17:38:15.000000000 -0800
+++ ../2012BOS.EVA	2014-11-19 17:36:57.000000000 -0800
@@ -9830,7 +9830,7 @@
 com,"Lance Barrett; when the Rangers asked about the call,"
 com,"crew chief Dana DeMuth watched the replay and upheld"
 com,"the call"
-play,9,0,cruzn002,32,1BFBFFB>B,W/UREV.1-2
+play,9,0,cruzn002,32,1BFBFFB>B,W/REV.1-2
 play,9,0,younm003,01,CX,8/F.2-3
 play,9,0,murpd005,12,BCFS,K
 play,9,1,gonza003,00,,NP
