--- 2012COL.EVN	2014-11-19 17:38:33.000000000 -0800
+++ ../2012COL.EVN	2014-11-19 17:39:04.000000000 -0800
@@ -2768,7 +2768,7 @@
 com,"$Jason Heyward hit a fly down the line that was ruled foul;"
 com,"when the Braves asked about the call, crew chief Tim"
 com,"McClelland watched the replay and upheld the call"
-play,7,0,heywj001,32,1CFBB*BB,W/UREV.1-2
+play,7,0,heywj001,32,1CFBB*BB,W/REV.1-2
 play,7,0,pastt001,00,2,PO2(E1).2-3;1-2
 play,7,0,pastt001,22,2.BFC*BFX,43/G
 play,7,0,martc005,00,,NP
