diff --git a/retrosheet/data/2013MIN.EVA b/retrosheet/data/../../../gamelog-data/retrosheet/event/regular/2013MIN.EVA
index 73aa15d..0a8e828 100644
--- a/retrosheet/data/2013MIN.EVA
+++ b/retrosheet/data/../../../gamelog-data/retrosheet/event/regular/2013MIN.EVA
@@ -5896,7 +5896,7 @@ play,7,1,escoe001,00,,NP
 sub,willj004,"Josh Willingham",1,8,11
 play,7,1,willj004,00,,NP
 sub,crowa001,"Aaron Crow",0,0,1
-play,7,1,willj004,01,..FX,DGR78/L.2-H;1-3
+play,7,1,willj004,01,..FX,DGR/L78.2-H;1-3
 play,7,1,florp001,00,,NP
 sub,dozib001,"Brian Dozier",1,8,12
 play,7,1,florp001,22,.SFBBS,K
