--- ../../../../Downloads/2014eve/2014ARI.EVN	2014-11-08 20:01:32.000000000 -0800
+++ 2014ARI.EVN	2014-12-17 17:30:49.000000000 -0800
@@ -2229,7 +2229,7 @@
 play,9,1,pradm001,00,,NP
 sub,papej001,"Jonathan Papelbon",0,9,1
 play,9,1,pradm001,01,.SX,S8/L
-play,9,1,goldp001,00,B,OA/MREV.1X2(24)
+play,9,1,goldp001,00,B,OA/REV.1X2(24)
 com,"replay,9,goldp001,ARI,davib902,PHO01,,Y,M,PHI,A"
 com,"$when Martin Prado was called safe at 2B, Phillies"
 com,"manager Ryne Sandberg challenged the ruling, saying"
