--- ../../../../Downloads/2014eve/2014DET.EVA	2014-11-25 07:21:46.000000000 -0800
+++ 2014DET.EVA	2014-12-17 17:34:45.000000000 -0800
@@ -2262,7 +2262,7 @@
 play,9,0,fowld001,00,,NP
 sub,nathj001,"Joe Nathan",1,0,1
 play,9,0,fowld001,32,.BSBBFFB,W
-play,9,0,castj006,02,CF*B,OA/UREV.1X2(26)
+play,9,0,castj006,02,CF*B,OA/REV.1X2(26)
 com,"replay,9,castj006,HOU,wintm901,DET05,,N,U,,A"
 com,"$when Dexter Fowler was called out at 2B, crew chief Mike Winters"
 com,"requested a review; the call was upheld by replay"
@@ -3933,7 +3933,7 @@
 play,9,0,reyej001,11,1C>B.X,S7/G.2-3
 play,9,0,cabrm002,00,>X,6/P
 play,9,0,bautj002,00,X,S8/G.3-H;1-3
-play,9,0,encae001,32,1BC1FF1B*B3,OA/MREV.1-2(TH3)
+play,9,0,encae001,32,1BC1FF1B*B3,OA/REV.1-2(TH3)
 com,"replay,9,encae001,TOR,gormb901,DET05,,N,M,DET,A"
 com,"$when Jose Reyes was called safe at 3B on a pickoff attempt,"
 com,"Tigers manager Brad Ausmus challenged the ruling; the call"
