--- ../../../../Downloads/2014eve/2014TBA.EVA	2014-11-08 20:01:28.000000000 -0800
+++ 2014TBA.EVA	2014-12-17 17:38:46.000000000 -0800
@@ -12130,7 +12130,7 @@
 play,6,0,younc004,22,CFBBFX,8/F
 play,6,1,lonej001,12,CBCFT,K
 play,6,1,frann001,12,BCSX,D7/L
-play,6,1,joycm001,32,SB*BBCFB,W+WP/MREV.2-3
+play,6,1,joycm001,32,SB*BBCFB,W+WP/REV.2-3
 com,"replay,6,joycm001,TBA,westj901,STP01,,Y,M,TBA,A"
 com,"$when Nick Franklin was called out at 3B, Rays"
 com,"manager Joe Maddon challenged the ruling; the"
