--- ../../../../Downloads/2014eve/2014MIN.EVA	2014-11-26 16:20:24.000000000 -0800
+++ 2014MIN.EVA	2014-12-17 17:38:04.000000000 -0800
@@ -66,7 +66,7 @@
 play,2,1,dozib001,32,FBBCF*B1>T,K+SB2
 play,2,1,mauej001,02,SCX,31/G
 play,3,0,fulds001,01,CX,63/G
-play,3,0,lowrj001,32,BCFFBBFB,W/UREV
+play,3,0,lowrj001,32,BCFFBBFB,W/REV
 com,"replay,3,lowrj001,OAK,millb901,MIN04,F,N,U,,H"
 com,"$Jed Lowrie hit a fly down the RF line on the 3-2 pitch"
 com,"that was ruled foul; crew chief Bill Miller requested a"
@@ -9430,7 +9430,7 @@
 play,5,1,mauej001,01,CX,63/G
 play,6,0,santc002,32,CCBBBF*B,W
 play,6,0,kipnj001,22,BC1CBS,K
-play,6,0,waltz001,12,BFS*B,OA/MREV.1X2(263)
+play,6,0,waltz001,12,BFS*B,OA/REV.1X2(263)
 com,"replay,6,waltz001,CLE,scotd901,MIN04,,N,M,CLE,I"
 com,"$when the pitch in the dirt was blocked by the catcher"
 com,"and Jason Kipnis was caught between the bases, Indians"
