--- ../../../../Downloads/2014eve/2014HOU.EVA	2014-11-08 20:01:18.000000000 -0800
+++ 2014HOU.EVA	2014-12-17 17:35:20.000000000 -0800
@@ -410,7 +410,7 @@
 play,9,0,cervf001,22,FBBFS,K
 play,9,0,robeb003,30,BBBB,W
 play,9,0,suzui001,12,CB1FF1FC,K
-play,9,0,solay001,31,PC1B*BB,W/UREV.1-2
+play,9,0,solay001,31,PC1B*BB,W/REV.1-2
 com,"replay,9,solay001,NYA,davig901,HOU03,,N,U,,K"
 com,"$crew chief Gerry Davis asked for a review of the pitches to"
 com,"determine the proper balls and strikes count; it was 3-1"
