--- ../../../../Downloads/2014eve/2014CHA.EVA	2014-11-17 08:38:08.000000000 -0800
+++ 2014CHA.EVA	2014-12-17 17:32:14.000000000 -0800
@@ -880,7 +880,7 @@
 play,1,0,swisn001,02,F1FS,K
 com,"$Felipe Paulino picked off Nyjer Morgan at 1B but Indians manager"
 com,"Terry Francona challenged the ruling; the call was overturned by replay"
-play,1,0,kipnj001,30,111BBBB,W/MREV.1-2
+play,1,0,kipnj001,30,111BBBB,W/REV.1-2
 com,"replay,1,kipnj001,CLE,davig901,CHI12,,Y,M,CLE,A"
 play,1,0,santc002,31,CBB*BB,W.2-3;1-2
 play,1,0,branm003,11,CBX,S8/L.3-H;2-H;1-3
@@ -11371,7 +11371,7 @@
 sub,putnz001,"Zach Putnam",1,0,1
 play,8,0,crisc001,31,.BSBBB,W
 play,8,0,fulds001,21,F1BB1X,7/F
-play,8,0,donaj001,12,1CFPF1*B,OA/MREV.1X2(24)
+play,8,0,donaj001,12,1CFPF1*B,OA/REV.1X2(24)
 com,"replay,8,donaj001,OAK,scord901,CHI12,,N,M,OAK,A"
 com,"$when Coco Crisp was called out at 2B, Athletics"
 com,"manager Bob Melvin challenged the ruling; the call"
