--- ../../../../Downloads/2014eve/2014TEX.EVA	2014-11-17 08:42:48.000000000 -0800
+++ 2014TEX.EVA	2014-12-17 17:39:08.000000000 -0800
@@ -11882,7 +11882,7 @@
 play,1,0,grosr001,32,CBCBBB,W
 play,1,0,altuj001,00,X,S7/L.1-2
 play,1,0,marij002,12,BSFS,K
-play,1,0,cartc002,01,S*B,OA/MREV.2X3(25);1-2
+play,1,0,cartc002,01,S*B,OA/REV.2X3(25);1-2
 com,"replay,1,cartc002,HOU,barrt901,ARL02,,Y,M,TEX,A"
 com,"$when Robbie Grossman was called safe at 3B, Rangers manager"
 com,"Tim Bogar challenged the ruling; the call was overturned"
